.travis.mk

```diff
@@ -453,7 +453,7 @@ build_freebsd:
 
 test_freebsd_no_deps: build_freebsd
 	make LuaJIT-test
-	cd test && python3.8 test-run.py --force $(TEST_RUN_EXTRA_PARAMS)
+	cd test && python3.8 test-run.py --vardir=/var/tmp/tt --force $(TEST_RUN_EXTRA_PARAMS)
```

Comment:
Nit: There is the `VARDIR` variable: AFAIS, we use it consistently across different targets for `--vardir`.

Reply:
For some reason VARDIR defined in environment/action.yml has no effect in FreeBSD. I will try to investigate the reason.

Reply:
test-run.py do not use VARDIR env variable at all. We can make it work like this:

```diff
% git diff lib/options.py 
diff --git a/lib/options.py b/lib/options.py
index 371b527..0fc95d6 100644
--- a/lib/options.py
+++ b/lib/options.py
@@ -13,6 +13,13 @@ def env_int(name, default):
     except ValueError:
         return default
 
+def env_str(name, default):
+    try:
+        value = os.environ.get(name)
+        return default if value is None else value
+    except ValueError:
+        return default
+
 
 def env_list(name, default):
     value_str = os.environ.get(name)
@@ -179,7 +186,7 @@ class Options(object):
         parser.add_argument(
                 "--vardir",
                 dest="vardir",
-                default="var",
+                default=env_str('VARDIR', "var"),
                 help="""Path to data directory. Default: var.""")
         parser.add_argument(
                "--long",
```

Reply:
Fixed with using variable instead of hardcoded string.